docs/streams/index.html

```diff
@@ -193,7 +193,7 @@ <h3 style="margin-top: 5.3rem;">Hello Kafka Streams</h3>
 
        <div class="code-example">
            <div class="btn-group">
-               <a class="selected b-java-8" data-section="java-8">Java 8+</a>
+               <a class="selected b-java-8" data-section="java-8">Java 11+</a>
```

Comment:
```suggestion
               <a class="selected b-java-8" data-section="java-8">Java</a>
```

Reply:
Not sure what `class="selected b-java-8" data-section="java-8"` is -- do you know?

Reply:
No, I also wondered what that is. :)